README.md

```diff
@@ -5,3 +5,5 @@
 This is an opinionated library to share code and approach to Datadog logging in prima.it
 
 Refer to the [official docs](https://docs.rs/prima_datadog) for help on how to setup the library in your project
+
+❕ Please note that `prima_datadog.rs` uses [DogstatsD](https://docs.datadoghq.com/developers/dogstatsd/), which means metrics will be sent using **UDP** on port `8125`. You can find more information on the official Datadog documentation
```

Comment:
```suggestion
❕ Please note that `prima_datadog.rs` uses [DogstatsD](https://docs.datadoghq.com/developers/dogstatsd/), which implements the [StatsD](https://github.com/statsd/statsd) protocol. This means that metrics will be sent using **UDP** on port `8125` (by default). You can find more information on the official Datadog documentation
```

Reply:
@MaeIsBad wdyt? I tried mentioning StatsD to be more precise

Reply:
Yeah that sounds good. Might be nice to provide an example of what we expect the configuration to be